src/index.html

```diff
@@ -192,6 +192,41 @@ <h3>About me</h3>
                 </div></li>
                 
             </ul>
-        </section>    
+        </section> 
+        <section class="footer">
+            <div class="title_form">
+               <h3>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+               </h3>
+               <form action="/my-handling-form-page" method="post">
+                <ul>
```

Comment:
- [ ] Please, consider, that `action` value should be similar to `https://formspree.io/f/{form_id}`. That is why your form does not submit correctly and gives an error:
<img width="481" alt="Screen Shot 2022-01-12 at 11 18 16" src="https://user-images.githubusercontent.com/49064106/149081558-fa2f74a1-30dc-4fc2-874d-b6119a705706.png">

Reply:
this is a frontend exercice i have no backend , this is only week 2 project, so would you explain what should i add to the action attribute in this case

Reply:
Sure I will support you. So first you should enter to https://formspree.io and create the account.  After, when you will get your id, just add it here in form action value add got id. Like this: `<form action="[/my-handling-form-page](https://formspree.io/f/{form_id})" method="post">`. Please, write if you have any questions.